Automattic/jetpack
# Add title attribute to the iframe for the regular archive.org embed

This commit adds a Title attribute to the iframe for the regular Archive.org book embed to improve the Section 508 and WCAG 2.0 accessibility compliance.

The change here tackles the open issue #7370 "Shortcode Embeds: should include title attribute on iframe output to meet accessibility standards" (There are multiple files which have iframes without a title attribute which are to be updated via separate PRs).

This change adds a fixed text string which would be a translatable string within Jetpack's i8ln language strings.

It also updates the test file `test-class.archiveorg.php` to include the new `title` attribute.

Fixes #

Changes proposed in this Pull Request:

A title attribute with a fixed i8ln string is added to the iframe that is rendered within this piece of code.

Is this a new feature or does it add/remove features to an existing part of Jetpack?

This adds to an existing feature of Jetpack.

Testing instructions:

To test this please try adding an Archive.org book using the archiveorg-book shortcode. E.g.

[archiveorg id=Wonderfu1958 width=320 height=240]

With the code in place the rendered iframe should now have a title attribute.

Proposed changelog entry for your changes:

This commit adds a Title attribute to the iframe for the regular Archive.org embed to improve the Section 508 and WCAG 2.0 accessibility compliance.
